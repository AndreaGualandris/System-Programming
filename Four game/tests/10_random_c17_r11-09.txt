5
13
7
7
7
11
11
5
15
11
5
11
4
8
14
2
16
1
5
10
15
1
6
2
6
7
2
16
6
1
2
0
12
15
13
12
13
15
8
5
10
11
12
16
7
8
6
10
14
3
14
16
5
0
1
9
15
6
6
9
4
2
5
8
10
7
14
2
0
12
12
16
3
11
9
12
11
4
11
14
16
2
4
5
8
6
16
6
5
1
14
12
16
11
11
10
4
5
4
7
15
5
14
11
2
3
2
14
11
14
11
1
6
16
1
14
0
0
16
1
2
5
4
2
16
16
13
13
1
2
8
7
4
0
11
2
15
11
2
4
5
3
1
6
5
14
9
16
11
5
0
13
1
3
4
15
12
9
14
14
16
2
12
0
1
0
10
10
4
7
15
15
7
1
8
6
11
1
8
15
7
11
4
15
1
3
7
