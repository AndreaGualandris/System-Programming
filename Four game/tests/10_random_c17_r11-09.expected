-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o           
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o       x   
-----------------
01234567890123456
Player 'o' to move:
       o
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o o     x   
-----------------
01234567890123456
Player 'x' to move:
       x
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
       x         
     o o     x   
-----------------
01234567890123456
Player 'o' to move:
       o
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o         
       x         
     o o     x   
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o         
       x         
     o o   x x   
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o         
       x   o     
     o o   x x   
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o         
     x x   o     
     o o   x x   
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o         
     x x   o     
     o o   x x o 
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
       o   x     
     x x   o     
     o o   x x o 
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     o o   x     
     x x   o     
     o o   x x o 
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
     o o   x x o 
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
    oo o   x x o 
-----------------
01234567890123456
Player 'x' to move:
        x
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
    oo ox  x x o 
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
    oo ox  x xoo 
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
  x oo ox  x xoo 
-----------------
01234567890123456
Player 'o' to move:
                o
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
  x oo ox  x xooo
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
           x     
     o o   x     
     x x   o     
 xx oo ox  x xooo
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
     o     x     
     o o   x     
     x x   o     
 xx oo ox  x xooo
-----------------
01234567890123456
Player 'x' to move:
          x
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
     o     x     
     o o   x     
     x x   o     
 xx oo ox xx xooo
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
     o     x     
     o o   x     
     x x   o   o 
 xx oo ox xx xooo
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
     o     x     
     o o   x     
 x   x x   o   o 
 xx oo ox xx xooo
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
     o     x     
     o o   x     
 x   x x   o   o 
 xx oooox xx xooo
-----------------
01234567890123456
Player 'o' wins!
